hadoop-ozone/dist/src/main/smoketest/compatibility/write.robot

```diff
@@ -68,7 +68,7 @@ File Can Be Deleted
 FSO Bucket Can Be Created and Used
     Pass Execution If    '${CLIENT_VERSION}' < '${FSO_VERSION}'    Client does not support FSO
     Pass Execution If    '${CLUSTER_VERSION}' < '${FSO_VERSION}'   Cluster does not support FSO
-    Execute    ozone sh bucket create --layout FILE_SYSTEM_OPTIMIZED /vol1/fso-bucket-${CLIENT_VERSION}
+    Execute    ozone sh bucket create --layout fso /vol1/fso-bucket-${CLIENT_VERSION}
```

Comment:
Not sure I understand... the CLI will be the new one right?

Reply:
It will be run if both client and cluster are 1.4.0 or later:

```
    Pass Execution If    '${CLIENT_VERSION}' < '${FSO_VERSION}'    Client does not support FSO
    Pass Execution If    '${CLUSTER_VERSION}' < '${FSO_VERSION}'   Cluster does not support FSO
```

So test covers eg. 1.4.0 client writing to 2.0.0 cluster, and 2.0.0 client writing to 1.4.0 cluster.  First one would fail with this.

Reply:
ah ok